exonum-java-binding-core/rust/Cargo.toml

```diff
@@ -19,7 +19,7 @@ exonum = "0.8.1"
 iron = "0.6.0"
 failure = "0.1.1"
 toml = "0.4.6"
-jni = { version = "0.10.1", git = "https://github.com/alexander-irbis/jni-rs", branch = "clone-jnienv", features = ["invocation"] }
+jni = { version = "0.10.1", git = "https://github.com/alexander-irbis/jni-rs", branch = "v0.10.1-exonum", features = ["invocation"] }
```

Comment:
It seems distinct from https://github.com/prevoty/jni-rs/pull/97 — does this branch have any extra patches?

Reply:
There is one another change: JNIEnv was made `Clone`able.